db/migrate/20190520224516_create_exercise_routines.rb

```diff
@@ -0,0 +1,14 @@
+class CreateExerciseRoutines < ActiveRecord::Migration[5.2]
+  def change
+    create_table :exercise_routines do |t|
+      t.integer :reps
+      t.integer :sets
+      t.integer :weight
+      t.integer :duration
+      t.references :routine, foreign_key: true
```

Comment:
So, this ensures that there will be a foreign key that references the routines table? 

and on the next line, a foreign key that references the exercises table?

And, for that matter, the test file on lines 25-26 of exercise_routines_spec?

Reply:
That's right!